### PR TITLE
Added DM01 Entryway and deathmatch PROMOD preset

### PR DESCRIPTION
**Pull request by Unisol**
Requires #31178
## About the PR

Added a (second) deathmatch map, based on a classic map from a different age. Added a deathmatch PROMOD preset (dmpm, deathmatchpromod), that works exactly like regular deathmatch but with guns.

## Why / Balance

One map is good, more is better. In addition, existing deathmatch map (Meteor Arena) is not very well fit for small arms combat, being designed for melee combat. PROMOD deathmatch is meant to replicate typical firefight that happens in SS14 but in deathmatch sandbox.

## Technical details

N/A

## Media

dm01-entryway-0

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

N/A

**Changelog**

:cl:
- add: added deathmatch map DM01 Entryway
- add: added deathmatch PROMOD preset

### CODE REVIEW

**Comment by KeldWolf** [comment]
This could be interesting for future Funday Sunday stuff. I can't imagine it facilitate very many players. Curious what others have to say about it.

**Comment by Unisol** [comment]
There's enough spawn points here for about 10 players, but it's hard to say how many can be comfortably packed in here. I'd guess about 5-7 people with guns can fit in this map without overcrowding.

File: Resources/Prototypes/Maps/arenas.yml [hunk 1]
```diff
@@ -12,3 +12,17 @@
...
+  minPlayers: 0
+  stations:
+    dm01-entryway:
+      stationProto: StandardStationArena
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: "DM01 Entryway"
+        - type: StationJobs
+          availableJobs:
+            Passenger: [ -1, -1 ]
```
Comment (FluffMe) [inline Resources/Prototypes/Maps/arenas.yml:28]: Please move this gameMap to `Resources/Prototypes/_Harmony/Maps/arenas.yml`

**Review by FluffMe — CHANGES_REQUESTED** [review]
One minor request to relocate.

**Comment by Unisol** [comment]
Issues addressed

**Comment by spanky-spanky** [comment]
Looks good, I’ll let Keld make the call to merge if we went this for Sunday Funday stuff but otherwise I approve.

**Review by FluffMe — APPROVED** [review]
Mostly admeme, noone objects. Merged

**Pull request merged**